src/coreclr/src/inc/corinfoinstructionset.h

```diff
@@ -28,8 +28,11 @@ enum CORINFO_InstructionSet
     InstructionSet_Sha1=8,
     InstructionSet_Sha256=9,
     InstructionSet_Atomics=10,
-    InstructionSet_Vector64=11,
-    InstructionSet_Vector128=12,
+    InstructionSet_Dp=11,
```

Comment:
Dp_Arm64 should also exist, it isn't covered by https://github.com/dotnet/runtime/pull/38460

Reply:
Okay, I will update/rebase this PR after you merge #38460

Reply:
Done